Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/participantsAdminService.ts

```diff
@@ -41,6 +41,18 @@ export const participantsAdminService = createApi({
                 body: participant,
             }),
         }),
+        exportParticipantsToExcel: builder.query<IFileModel, IGetAllAdminParams>({
+            query: ({ filter, page, itemsPerPage, sorting }) => ({
+                url: `/${EXCEL_RESULTS_ENDPOINT}`,
+                params: {
+                    filter,
+                    page,
+                    itemsPerPage,
+                    sorting,
+                },
+            }),
+            keepUnusedDataFor: 5,
```

Comment:
Why here is 5 and on other places 0 or 10?

Reply:
Fixed